### DISCUSSION TITLE
sunset_time override not working as expected

### DISCUSSION THREAD

**Discussion by pettertho** [basnijholt/adaptive-lighting#204] [Q&A]
When inputting the sunset_time override value to replace "None" I insert e.g. "22:30:00", but if I do that in the evening, not long before that time, the brightness goes to 100%, which does not make sense.

The guide text does not page break which causes me to not be able to read if my input value is invalid or there is something else I am doing wrong. 

Expected result is that sunset happens at sunset_time override value, and if I enable the given value around 22:00:00, I should expect brightness to be much lower than 100%, given default settings elsewhere.

Are you able to give me any pointers as to what I am doing wrong? :)

**Comment by pesbob** [comment]
I have the same problem.

**Comment by th3w1zard1** [comment]
Please post your adaptive-lighting config and setup your logbook to track the state of the brightness_pct attribute so we can see what's going on.
